packages/debugger/lib/web3/adapter.js

```diff
@@ -10,7 +10,8 @@ export default class Web3Adapter {
   }
 
   async getTrace(txHash) {
-    let result = await promisify(this.web3.currentProvider.send)(
+    const provider = this.web3.currentProvider;
+    const result = await promisify(provider.send.bind(provider))(
```

Comment:
Might be worth leaving a comment explaining the need for the bind.

Reply:
Yeah, not a bad idea.

Reply:
That works!